dydx-stark/src/endpoint/starkex.ts

```diff
@@ -19,12 +19,19 @@ export type PriceStarkPayload = PriceDataPoint & {
 
 const MAX_DECIMALS = 18
 
-const ZERO_BN = new BN('0')
-const TWO_BN = new BN('2')
-const TEN_BN = new BN('10')
+const ZERO_BN = BigNumber.from('0')
+const TWO_BN = BigNumber.from('2')
+const TEN_DEC = new Decimal('10')
```

Comment:
There is no reason to use decimals here, as I do all calculations with big numbers. I convert from decimal to big number manually first, and then all further calculations are BN.